tests/test_utils.py

```diff
@@ -742,18 +742,7 @@ def clean_up_tables_with_prefix(connection_string: str, table_prefix: str) -> Li
     execution_engine: SqlAlchemyExecutionEngine = SqlAlchemyExecutionEngine(
         connection_string=connection_string
     )
-    data_connector = instantiate_class_from_config(
-        config={
-            "class_name": "InferredAssetSqlDataConnector",
-            "name": "temp_data_connector",
-        },
-        runtime_environment={
-            "execution_engine": execution_engine,
-            "datasource_name": "temp_datasource",
-        },
-        config_defaults={"module_name": "great_expectations.datasource.data_connector"},
-    )
-    introspection_output = data_connector._introspect_db()
+    introspection_output = introspect_db(execution_engine=execution_engine)
```

Comment:
This code previously used an `InferredAssetSqlDataConnector` to introspect the database and obtain table names for the test teardown process. `data_connector._introspect_db()` had no dependencies on the data connector class outside of the `SqlAlchemyExecutionEngine`, so I refactored that method to a stand alone function which requires the execution engine as an argument.